requirements-docs.txt

```diff
@@ -0,0 +1,2 @@
+Sphinx==1.1.3
+-rrequirements.txt
```

Comment:
Not sure I follow.  My issue was more that we're always building docs against the develop version of botocore instead of whatever the setup.py says you should use for botocore.  This means that `requirements-docs.txt` should *not* be used for anything other than the develop branch or the latest tag.  Otherwise, something like this would be wrong:

```
git checkout 1.9.0
pip install -r requirements-docs.txt
cd docs && make html
```